packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -161,6 +173,12 @@ export interface BaseServiceOptions {
    * @default - Rolling update (ECS)
    */
   readonly deploymentController?: DeploymentController;
+
+  /**
+   * Whether to enable the deployment circuit breaker
+   * @default - disabled
+   */
+  readonly circuitBreaker?: DeploymentCircuitBreaker;
```

Comment:
I see in line 380 that we are always setting "enable" to be true, which makes this comment about the default setting a little confusing. 

If the `DeploymentCircuitBreaker` only contains the boolean `rollback`, would it make sense then to just have this is a boolean field on `BaseServiceOptions`?

Reply:
@SoManyHs  OK Let me do it now.

Reply:
Just updated the description. Let me know if it looks better now.

@SoManyHs

Reply:
@SoManyHs as for now it just has `rollback` but yes potentially in the future they'll add more options to it.